tests/functional/disaster-recovery/metro-dr/test_discovered_vms_failover_and_relocate.py

```
@@ -0,0 +1,500 @@
+                id="primary_down-recipe_with_checkhooks",
+            ),
+        ],
+    )
+    def test_discovered_vm_failover_and_relocate(
+        self,
+        primary_cluster_down,
+        recipe,
+        checkhooks,
+        discovered_apps_dr_workload_cnv,
+        nodes_multicluster,
+        node_restart_teardown,
```

Ruff (0.15.12)

[warning] Unused method argument: `node_restart_teardown`

(ARG002)